.circleci/config.yml

```diff
@@ -104,17 +99,17 @@ workflows:
   build:
     jobs:
 
-      - test-2.3-with-4.2
-      - test-2.3-with-5.0
-      - test-2.3-with-5.1
-      - test-2.3-with-5.2
-
       - test-2.4-with-4.2
-      - test-2.4-with-5.0
       - test-2.4-with-5.1
       - test-2.4-with-5.2
 
       - test-2.5-with-4.2
-      - test-2.5-with-5.0
       - test-2.5-with-5.1
       - test-2.5-with-5.2
+
+      - test-2.6-with-4.2
+      - test-2.6-with-5.1
+      - test-2.6-with-5.2
+
+      - test-2.7-with-5.1
```

Comment:
No `- test-2.7-with-4.2` was deliberate right?

Reply:
Yes. Rails 4.2 does not work with the newest Ruby.